packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebViewClient.java

```diff
@@ -30,6 +34,47 @@
     this.methodChannel = methodChannel;
   }
 
+  private static String errorCodeToString(int errorCode) {
+    switch (errorCode) {
+      case WebViewClient.ERROR_AUTHENTICATION:
+        return "AUTHENTICATION";
+      case WebViewClient.ERROR_BAD_URL:
+        return "BAD_URL";
+      case WebViewClient.ERROR_CONNECT:
+        return "CONNECT";
+      case WebViewClient.ERROR_FAILED_SSL_HANDSHAKE:
+        return "FAILED_SSL_HANDSHAKE";
+      case WebViewClient.ERROR_FILE:
+        return "FILE";
+      case WebViewClient.ERROR_FILE_NOT_FOUND:
+        return "FILE_NOT_FOUND";
+      case WebViewClient.ERROR_HOST_LOOKUP:
+        return "HOST_LOOKUP";
+      case WebViewClient.ERROR_IO:
+        return "IO";
+      case WebViewClient.ERROR_PROXY_AUTHENTICATION:
+        return "PROXY_AUTHENTICATION";
+      case WebViewClient.ERROR_REDIRECT_LOOP:
+        return "REDIRECT_LOOP";
+      case WebViewClient.ERROR_TIMEOUT:
+        return "TIMEOUT";
+      case WebViewClient.ERROR_TOO_MANY_REQUESTS:
+        return "TOO_MANY_REQUESTS";
+      case WebViewClient.ERROR_UNKNOWN:
+        return "UNKNOWN";
+      case WebViewClient.ERROR_UNSAFE_RESOURCE:
+        return "UNSAFE_RESOURCE";
+      case WebViewClient.ERROR_UNSUPPORTED_AUTH_SCHEME:
+        return "UNSUPPORTED_AUTH_SCHEME";
+      case WebViewClient.ERROR_UNSUPPORTED_SCHEME:
+        return "UNSUPPORTED_SCHEME";
+    }
+
+    final String message =
+        String.format(Locale.getDefault(), "Could not find a string for errorCode: %d", errorCode);
```

Comment:
nit:

Is `%d` formatting affected by the locale?
Also seems like you can omit specifying `Locale.getDefault()` as this is the default for the `String.format` version that doesn't take a locale.

Reply:
I only added the `Locale.getDefault()` to get rid of the lint warning. It does change the digit to whatever locale the device is set to. I don't think it would change the interpretation of the digit tho.

Reply:
Interesting, my read of https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html#syntax is that `%d` formatting is not affected by the locale.